tests/test_io_utils.py

```diff
@@ -30,11 +31,13 @@ def test_img_read(input_folder: str, filename: str, mean: float, shape: Tuple[in
     assert img.flags["WRITEABLE"], "Obtained numpy array is not writeable"
 
 
-xml_root = ET.Element("EOPatch")
-xml_data = ET.SubElement(xml_root, "data")
-ET.SubElement(xml_data, "field1", name="BANDS-S2-L1C").text = "some value1"
-ET.SubElement(xml_data, "field2", name="CLP").text = "some value2"
-TEST_XML = ET.ElementTree(xml_root)
+@pytest.fixture
+def xml_test():
```

Comment:
i'd go with `xml_testcase` or `xml_data`, just so it doesn't carry a connotation of being a test itself

Reply:
done